src/app/globals.css

```diff
@@ -10,3 +12,33 @@
   color: var(--primary-font);
   box-sizing: border-box;
 }
+
+h1 {
+  font-style: normal;
+  font-weight: 800;
+  font-size: 22px;
+}
+
+h2 {
+  font-size: 56px;
+  color: var(--h2-font);
+  font-weight: 800;
+
+  @media (max-width: 768px) {
+    font-size: 40px;
+  }
+
+  @media (max-width: 281px) {
```

Comment:
all your media queries should be consistent values like 375, 768, 1024, 1440, etc. using arbitrary values makes it too hard to test all your scenarios.

Try to keep all your breakpoints consistent for example on a complex website you'd do something like: 
1. I have a layout for 1px => 767px (mobile)
2. 768px => 1024px (tablet)
3. 1024px+ (desktop)
4. 1920px+ (massive desktop) sometimes this is needed but rarely

For a small website like Sams, it would be best to keep it simple with two breakpoints. this is important because then when testing changes, you only really have to test two screen sizes. otherwise its too hard to test if you use arbitrary values everywhere. it'll be true the website might not be perfect on some random screen size, but thats okay because it saves us work and 95% of users will either be 350px - 450px for mobile or 1024px-2000px for desktop. I would stick with these two for sams:

1. Mobile layout: 1px to 768 or maybe 1023px
2. Desktop layout:  1024px+

And of course, there may be edge cases to this where you do need a specific breakpoint for a component and that's fine but for 95% of what we build, we want to stick to these two to make it simple

Reply:
Yeah this makes sense. I had looked up some media queries for tables and I read 990 s0 thats why I had that one. And then 281 was for Galaxy Fold since that one looked bad. I got rid of them though and replaced with 1023px so mobile and tablet will be lumped together.